builtin/providers/aws/resource_aws_cloudtrail.go

```diff
@@ -22,6 +22,11 @@ func resourceAwsCloudTrail() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"enable_logging": &schema.Schema{
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
```

Comment:
You can keep the field name, just change this to `Default: true` and it should be good to go. :smiley: